frontend/src/component/changeRequest/ChangeRequestPermissions.test.tsx

```diff
@@ -119,6 +127,15 @@ const userHasPermissions = (permissions: Array<IPermission>) => {
         splash: {},
     });
 };
+const userIsMemberOfProjects = (projects: string[]) => {
+    userHasPermissions(
```

Comment:
we detect who is a project member by filtering auth user permissions and looking for any mention of a given project